app/assets/stylesheets/application.scss

```diff
@@ -125,3 +127,18 @@ a:hover {
 .panel #created {
   color: $font-color;
 }
+
+.field label {
+  color: $font-color;
+  font-size: 1em;
+}
+
+.tag {
+  border-style: solid;
```

Comment:
Properties should be ordered border-color, border-radius, border-style, border-width, padding, text-align